package.json

```diff
@@ -176,7 +176,7 @@
     "styled-components": "^5.1.1",
     "valid-url": "^1.0.9",
     "zealot": "file:zealot",
-    "zq": "git+https://github.com/brimsec/zq.git#fa21f47928a47b60df94007dda0a9fadffd2440a"
+    "zq": "git+https://github.com/brimsec/zq.git#v0.19.0"
```

Comment:
```suggestion
    "zq": "git+https://github.com/brimsec/zq.git#v0.23.0"
```

Reply:
(and similarly for package-lock.json)

Reply:
@alfred-landrum: Thanks for catching that mistake. I've pushed the correction and will make sure the automation passes.